### PR TITLE
[DASH-2237] Add hosting name in Inner toolbar component

### PR DESCRIPTION
**Pull request by Flixow**

### CODE REVIEW

**Comment by tomsonpl** [comment]
LGTM

**Comment by patrykkopycinski** [comment]
@Flixow screenshot?

File: src/apps/Hosting/Files.js [hunk 1]
```diff
@@ -76,17 +75,26 @@ const HostingFilesView = React.createClass({
...
+      currentFileIndex,
+      isUploading
+    } = this.state;
     const hasFilesToUpload = filesToUpload.length > 0;
     const isDefaultHosting = this.isDefaultHosting();
+    const hostingLabel = hostingDetails ? hostingDetails.label : '';
 
     return (
       <div>
-        <Helmet title="Website Hosting" />
```
Comment (patrykkopycinski) [inline src/apps/Hosting/Files.js]: why did you remove it?

**Comment by Flixow** [comment]
image

File: src/apps/Hosting/Files.js (hunk 1 above)
Comment (Flixow) [inline src/apps/Hosting/Files.js]: My bad :/

File: src/apps/Hosting/Files.js [hunk 2]
```diff
@@ -76,17 +76,27 @@ const HostingFilesView = React.createClass({
...
+        <Helmet title={`Website Hosting: ${hostingLabel}`} />
```
Comment (patrykkopycinski) [inline src/apps/Hosting/Files.js]: ```{`Website Hosting: ${hostingLabel}`}``` move into variable
